docs/docs/segment-npm.md

```diff
@@ -0,0 +1,48 @@
+---
+id: npm
+title: npm
+sidebar_label: NPM
+---
+
+## What
+
+Display the currently active npm version.
+
+## Sample Configuration
+
+```json
+{
+  "type": "npm",
+  "style": "powerline",
+  "powerline_symbol": "\uE0B0",
+  "foreground": "#193549",
+  "background": "#ffeb3b",
+  "properties": {
+    "template": "{{.NpmIcon}} {{ .Full }} "
+  }
+}
+```
+
+## Properties
+
+- npm_icon: `string` - the icon/text to display - defaults to `\ue71e`
```

Comment:
You could actually make this part of the template as it's completely static and the configuration change is identical (but removed logic so is more efficient).

Reply:
Changes coming in